constructor/main.py

```diff
@@ -109,6 +109,12 @@ def main_build(dir_path, output_dir='.', platform=cc_platform,
         if info.get(key):  # only join if there's a truthy value set
             info[key] = abspath(join(dir_path, info[key]))
 
+    # Normalize name and set default value
+    if info.get("windows_signing_tool"):
+        info["windows_signing_tool"] = info["windows_signing_tool"].lower().replace(".exe", "")
```

Comment:
At the same time, if we are validating the input in construct.py:805, why do we need normalization here? 🤔 For library usage? In that case it should be the same logic (or move the logic here).

Reply:
> What happens if a user tries to set this to a full path? 

`constructor` will and should fail. `windows_signing_tool` is the name. Otherwise, I would have to infer from the binary what the appropriate signing too is.

>  In that case it should be the same logic

Yes, that's true, good catch!

> (or move the logic here).

I'm not sure about this. Setting default values and manipulating input is done in `main_build`. `validate()` performs validation here. If we move this line into `validate()`, we are mixing scopes, which makes things harder to track.